website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md

```diff
@@ -0,0 +1,640 @@
+
+_Note: This approach will change with the new Semantic Layer connection, which will be able to query all data platforms supported in dbt Cloud via dedicated JDBC/ODBC drivers (and eventually an API)._
+
+
+<!--- tabs for running models and running metrics --->
+<Tabs>
+
+<TabItem value="metrics" label="Metric queries">
+
+Users can compile and execute metric queries using macros defined in the [dbt-metrics package](https://github.com/dbt-labs/dbt_metrics).  This package:
+
+* Generates the SQL required to accurately calculate the metric definition,
+* Supplies helper macros for derived calculations (like month over month, year to date, and so on) time series operations
+
+
+```
+select * 
+from {{ metrics.calculate(
+    metric_list=[metric('customers'), metric(‘revenue’)],
+    grain='week',
+    dimensions=['plan', 'country'],
+    secondary_calculations=[
+        metrics.period_to_date(aggregate="sum", period="year"),
+        metrics.rolling(aggregate="average", interval=4, alias="avg_past_4wks")
+    ],
+    start_date='2020-01-01',
+    end_date="date_trunc('day', getdate())"
+) }}
+```
+
+</TabItem>
+
+<TabItem value="models" label="Model queries">
+
+Model queries allow users to query models and use macros from their dbt project.
+
+```
+select cents_to_dollars('amount_cents') as amount_dollars 
+from {{ ref('orders') }}
+```
+</TabItem>
+</Tabs>
+
+## Product Roadmap
+
+The dbt Semantic Layer product roadmap details what features are coming soon and the timelines associated with them. 
+
+| Product | Estimated launch |
+| ------- | ---------------- |
+| [Entities](#entities) | Early 2023 |
+| [dbt Semantic Layer Connector](#dbt-semantic-layer-connector) | Early-mid 2023 |
+| [dbt Semantic Layer API](#dbt-semantic-layer-api) | Mid 2023 |
+
+### Entities 
+<!-- rewrite a little and include a summary for each tab entry -->
+
+dbt Labs will introduce a new node type, [the entity](https://github.com/dbt-labs/dbt-core/issues/6379) when dbt Core version 1.5 launches. It introduces a new and efficient way to define metrics by reusing logic (for example, `time_grains`).  Entities are semantic objects made up of curated dimensions from models with more metadata defined. Over time, users can standardize metric and entity definitions with packages to speed up development. 
+
+For integrations, entities will provide information like:
+
+- a way to organize metrics based on the entity they reference, and 
+- a new consumable and dynamically generated dataset (versus finding a table in the data platform). 
+
+This information will be available alongside the Metadata API and entities can be directly queried through the dbt Semantic Layer. 
+
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/entity-lineage.jpg" title="Entity lineage graph" />
+
+:::caution 🚧
+
+This is a work in progress and you can anticipate continuous improvements. 
```

Comment:
@callum-mcdata can i link to a place where users can stay up to date w entities? if so, is [this](https://github.com/dbt-labs/dbt-core/issues/6379) the link for that?

```suggestion
This is a work in progress and you can anticipate continuous improvements. 
```

Reply:
@mirnawong1 yea that would be the best place for keeping up to date for now. In the future it might contain links to the suggested PR's once the code is in draft form.